src/frontend/Sider/Personoversikt/Dokumentoversikt/DokumentTabell.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Table } from '@navikt/ds-react';
+
+import DokumentRad from './DokumentRad';
+import { Underrad } from './Underrad';
+import { DokumentInfo } from '../../../typer/dokument';
+import { groupBy } from '../../../utils/utils';
+
+export const DokumentTabell: React.FC<{ dokumenter: DokumentInfo[] }> = ({ dokumenter }) => {
+    const dokumentterGrupperPåJournalpost = groupBy(
+        dokumenter,
+        (dokument) => dokument.journalpostId
+    );
+
+    const journalposterSortertPåTid = Object.keys(dokumentterGrupperPåJournalpost).sort(
+        function (a, b) {
+            const datoA = dokumentterGrupperPåJournalpost[a][0].dato;
+            const datoB = dokumentterGrupperPåJournalpost[b][0].dato;
+            if (!datoA) {
+                return -1;
+            } else if (!datoB) {
+                return 1;
+            }
+            return datoA > datoB ? -1 : 1;
+        }
+    );
```

Comment:
Kan vurdere om det er enklere å bare sette sortering fra backend til `.sortedByDescending { it.dato }` 😁 
Eller/og at **grupperingen** gjøres i backend

Reply:
Enig at de burde vært sånn, og gjerne med litt tester også. Men prioriterer ikke det nå.